phpstan-baseline.neon.dist

```diff
@@ -85,16 +85,6 @@ parameters:
 			count: 1
 			path: system/Cache/Handlers/MemcachedHandler.php
 
-		-
```

Comment:
I was getting errors since these configs are in place but no matching error was found.